CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 16.8.3
+
+- Fixed: `color-function-notation` reporting functions with less than 5 arguments ([#7948](https://github.com/stylelint/stylelint/pull/7948)) ([@Mouvedia](https://github.com/Mouvedia)).
```

Comment:
```suggestion
- Fixed: `color-function-notation` reporting functions with less than 3 arguments ([#7948](https://github.com/stylelint/stylelint/pull/7948)) ([@Mouvedia](https://github.com/Mouvedia)).
```

5 is the number for us developers.
3 doesn't account the commas, which is much comprehensible for our users.